pkg/controller/config/config_controller.go

```diff
@@ -251,7 +251,7 @@ func (r *ReconcileConfig) Reconcile(req reconcile.Request) (reconcile.Result, er
 func (r *ReconcileConfig) applyPipeline(req reconcile.Request, cfg *op.Config) (reconcile.Result, error) {
 	log := requestLogger(req, "apply-pipeline")
 
-	if err := transformManifest(cfg, &r.pipeline); err != nil {
+	if err := transformManifest(cfg, &r.pipeline, transform.ReplaceDeploymentImages("IMAGE_PIPELINE_")); err != nil {
```

Comment:
shall we move `IMAGE_PIPELINE_` prefix string into a `config` in `pkg/flag/flag.go`